x/mint/keeper/keeper_test.go

```diff
@@ -59,6 +63,16 @@ func TestKeeperTestSuite(t *testing.T) {
 	suite.Run(t, new(KeeperTestSuite))
 }
 
+func (suite *KeeperTestSuite) TestGRPCParams() {
```

Comment:
Can we still keep this in `grpc_query_test.go` please?

I think the goal is to share the suite - `KeeperTestSuite` but keep the file locations preserved.

In Go, the convention is to have the test for a method/function in the file with the same name, suffixed by `_test.go`. Since the Params query is in the `grpc_query.go`, we should keep having `TestGRPCParams` where it used to be.

`genesis_test.go` shares the test suite with the keeper:
https://github.com/osmosis-labs/osmosis/blob/05375f8603e4d2e294281df267ee891405a32b7c/x/mint/keeper/genesis_test.go#L42-L44

We should do something similar for the grpc query tests

Reply:
aah i see. That makes sense! i'll change it rn